<launch>

  <!-- Upload capsule collision operations  & description-->
  <include file="$(find capsule_collision)/launch/reemc_capsule_description.launch" />

  <!-- Configure wbc -->
  <rosparam command="load"
    file="$(find reemc_wbc_tutorials)/config/reemc_wbc.yaml" />
  
  <!-- Floating base transform publisher -->
  <node name="floating_base_publisher"
        pkg="pal_wbc_controller" type="floating_base_publisher" />

  <!-- Spawn controller -->
  <node name="wbc_spawner"
        pkg="controller_manager" type="spawner" output="screen"
        args="whole_body_kinematic_controller" />
        
</launch>
